src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/WindowsSettings.schema.json

```diff
@@ -0,0 +1,76 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "additionalProperties": false,
+    "required": [ "settings" ],
+    "properties": {
+        "settings": {
+            "type" : "array",
+            "items": {
+                "additionalProperties": false,
+                "required" : [ "Name", "Command", "Type" ],
+                "type":"object",
+                "properties": {
+                    "Name": { "type":"string" },
+                    "Areas": {
+                        "type":"array",
+                        "items": {
+                            "type": "string",
+                            "enum": [
+                                "AreaAccounts",
+                                "AreaAdministrativeTools",
+                                "AreaApps",
+                                "AreaAppearanceAndPersonalization",
+                                "AreaClockAndRegion",
+                                "AreaCortana",
+                                "AreaDevices",
+                                "AreaEaseOfAccess",
+                                "AreaExtras",
+                                "AreaGaming",
+                                "AreaHardwareAndSound",
+                                "AreaHomePage",
+                                "AreaMixedReality",
+                                "AreaNetworkAndInternet",
+                                "AreaPersonalization",
+                                "AreaPhone",
+                                "AreaPrivacy",
+                                "AreaPrograms",
+                                "AreaSurfaceHub",
+                                "AreaSystem",
+                                "AreaSystemAndSecurity",
+                                "AreaTimeAndLanguage",
+                                "AreaUserAccounts",
+                                "AreaUpdateAndSecurity"
+                            ]
```

Comment:
Do we really want to have this? This needs to be updated on changes in the settings app.

What happened if we have a value that is not in the list? Do we check the json against the scheme when reading it?

Reply:
This was only an idea, Yes you are right, i will change it to the suggestion in your comment before.

Reply:
I think we also should not have the enum on `Type` and `DepreciatedIn` and `IntroducedIn`.